apps/studio-client/src/main/joo/com/coremedia/labs/translation/gcc/studio/GccProcessDefinitions_de.properties

```diff
@@ -69,6 +69,6 @@ ErrorCode_GCC_WF_40001_text=Die Konfiguration der GlobalLink-Verbindung ist fehl
 ErrorCode_GCC_WF_40050_text=Die GlobalLink-Auftragsnummer ist fehlerhaft.
 ErrorCode_GCC_WF_50050_text=Der XLIFF-Export ist fehlgeschlagen.
 ErrorCode_GCC_WF_61001_text=Das Abbrechen des GlobalLink-Auftrags ist fehlgeschlagen.
-Action_Tooltip_Cancel_Process=Den gewählten Prozess abbrechen
+Action_Tooltip_Cancel_Process=Die gewählte Übersetzung mit GlobalLink abbrechen
```

Comment:
Please ensure you have native to ASCII conversion enabled. We must not add non-ASCII characters here, as it may break on some platforms.